### DISCUSSION TITLE
need some more informations to improve my settings

### DISCUSSION THREAD

**Discussion by AlexCGN13** [CodyBerenson/PGMA-Modernized#299] [Q&A] [answered]
Hi Cody, 
Hi All , 
@Cody: thank You so much for creating / improving those wonderful agents. 
As I am new to it , I read the manual and am now trying around before indexing my bigger collection of porn: 

<img width="335" alt="screen shot" src="https://github.com/CodyBerenson/PGMA-Modernized/assets/41262722/4ef8e348-b1ce-4eb2-80c9-708338724043">
I still  have some questions about the setting: 

first is about duration difference: if  I set tolerated duration difference to 1 ,does it mean that the tolerated duration difference is 1% ?

the second question is about "choose Poster Source, and whether to download to Disk: "

Local -yes, local -no ,  External - yes, External - no

Do I need to set to external  - yes , unless Plex won't show any covers ? 

Sorry for my poor English , which might be the reason of my question :) 

Thanks for your Help+ Best Regards, 

Alex

**Comment by JPH71** [comment, marked as answer]
The duration is not a percentage... its difference in minutes

The download of posters to disk is for some users... they wanted the
posters to add as metadata in their files.. scraping will still work
whether you have this set or not.

If you have any problems whilst scraping, ask and one if us will usually
answer within a day

Happy scraping!!

Jason

**Comment by CodyBerenson** [comment]
@AlexCGN13 ,

Thank you for your kind words.  The truth is that @JPH71 is the brains behind the agents.  I am merely the exceptional eye candy.  Cheers!

**Comment by JPH71** [comment]
Damn! You nailed that one Cody!
